docs/specs/markdown.yml

```diff
@@ -80,6 +80,20 @@ outputs:
   build.log: |
     ["warning","heading-not-found","The first visible block is not a heading block with `#`","docs/a.md"]
 ---
+# Include block before h1
+inputs:
+  docfx.yml: |
+    rules:
+      heading-not-found: warning
+  docs/a.md: |
+    [!include[]()]
+    # Heading
+outputs:
+  docs/a.json:
```

Comment:
verify h1 remained in the output?

Reply:
because there is a visible block before h1. We only extract the h1 as title if it is the first visible block